src/ext/ca/serverca/scasched/scauser.cpp

```diff
@@ -540,6 +540,8 @@ HRESULT ScaUserExecute(
             ueUserExists = USER_EXISTS_INDETERMINATE;
             hr = HRESULT_FROM_WIN32(er);
             WcaLog(LOGMSG_VERBOSE, "Failed to check existence of domain: %ls, user: %ls (error code 0x%x) - continuing", wzDomain, psu->wzName, hr);
+			hr = S_OK;
+			er = ERROR_SUCCESS;
```

Comment:
Right. I'm commenting that the whitespace appears to be incorrect. Were tabs used instead of spaces or is GitHub just rendering it funny?